docs/sql-programming-guide.md

```diff
@@ -1022,6 +1156,117 @@ results = sqlContext.sql("FROM src SELECT key, value").collect()
 </div>
 </div>
 
+## JDBC Connectivity
+
+Spark SQL also includes a datasource that can read data from other databases using JDBC.  This
+functionality is similar to JDBCRDD, but since the results are returned as a DataFrame they can
```

Comment:
I actually think for SEO we want to.  If people are searching JdbcRDD then I'd love them to end up here.  Do you think I should say that its going to be deprecated?